### ISSUE TITLE
Crash when WebView2 is blocked by group policy

### ISSUE THREAD

**Issue by cvlnomen** [Kizari/Flagrum#1]
When the user has WebView2/Microsoft Edge disabled with group policy, Flagrum will immediately crash after startup without giving any information about the problem. It would be better if Flagrum first checks if WebView2 is accessible. If this is not the case, the program should show a popup that tells the user what the problem could be.

Here is the crash log that you get when Edge is disabled:

```
System.AggregateException
Message: A Task's exception(s) were not observed either by Waiting on the Task or accessing its Exception property. As a result, the unobserved exception was rethrown by the finalizer thread. (This program is blocked by group policy. For more information, contact your system administrator. (0x800704EC))
Stack Trace: 
System.Runtime.InteropServices.COMException
Message: This program is blocked by group policy. For more information, contact your system administrator. (0x800704EC)
Stack Trace:    at System.Runtime.InteropServices.Marshal.ThrowExceptionForHR(Int32 errorCode)
   at Microsoft.Web.WebView2.Core.CoreWebView2Environment.CreateCoreWebView2ControllerAsync(IntPtr ParentWindow)
   at Microsoft.Web.WebView2.Wpf.WebView2.<>c__DisplayClass27_0.<<EnsureCoreWebView2Async>g__Init|0>d.MoveNext()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Components.WebView.WebView2.WebView2WebViewManager.InitializeWebView2()
   at Microsoft.AspNetCore.Components.WebView.WebView2.WebView2WebViewManager.<>c__DisplayClass4_0.<<NavigateCore>b__0>d.MoveNext()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Components.WebView.Wpf.WpfDispatcher.InvokeAsync(Func`1 workItem)
   at Microsoft.AspNetCore.Components.WebView.Wpf.WpfDispatcher.<>c.<.cctor>b__8_0(Exception exception)
```
You can use the following program to reproduce this issue: https://www.sordum.org/9312/edge-blocker-v1-7/ 

I also created PR #2 that adds a message box when the exception occurs. I could not find a reliable way to check if WebView2 is blocked by the group policy.

**Comment by Kizari** [comment]
I hadn't really considered that some people may be blocking Edge and thus also by proxy WebView2. I think a MessageBox is a suitable enough solution given the circumstances, and like you say the most important part is just to inform the user of _why_ the crash is occurring so that they can decide whether or not it's worth unblocking edge to use the software.

Thank you for this. I'll merge the PR when the next update is being done.

**Comment by Kizari** [comment]
**Developer Note**
With the localisation branch ready to be merged in, a Japanese version of this MessageBox should also be added to the PR after merging.

**Comment by Kizari** [comment]
PR has been merged and localised. Will be released as part of Flagrum 1.2.5 in the near future. Thanks again for the contribution.
